Sources/BraintreeCore/BTConfiguration.swift

```diff
@@ -15,6 +15,18 @@ import Foundation
         json?["environment"].asString()
     }
     
+    /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    /// The Braintree GW URL to use for REST requests
+    public var clientAPIURL: URL? {
```

Comment:
Could `clientAPIURL` and `graphQLURL` be internal or are the used outside of core?